packages/openapi3/src/lib.ts

```diff
@@ -27,24 +27,28 @@ export const libDef = {
         default: "Duplicate @body declarations on response type",
       },
     },
+    // TODO-REMOVE moved to rest lib
```

Comment:
Why can't we remove these straight away?

Reply:
was just a todo for myself, removing now. I think I also need to move all the test that were testing those diagnostics into the rest lib now.